modules/Microsoft.Network/publicIPPrefixes/readme.md

```diff
@@ -15,7 +15,7 @@ This template deploys a public IP prefix.
 | :-- | :-- |
 | `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
 | `Microsoft.Authorization/roleAssignments` | [2020-10-01-preview](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2020-10-01-preview/roleAssignments) |
-| `Microsoft.Network/publicIPPrefixes` | [2021-05-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-05-01/publicIPPrefixes) |
+| `Microsoft.Network/publicIPPrefixes` | [2022-01-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/publicIPPrefixes) |
```

Comment:
Please update to latest version documented in ARR so that the link is mapped to the specific version used and not to generic latest (which can change over time)

Reply:
Updated to older API version